main.py

```diff
@@ -121,6 +91,17 @@ def lookup_query(self, query):
             return {term: self.index[term] for term in from_doc_to_terms(query) if term in self.index}
         else:
             return {term: self.index[term] for term in query.split(' ') if term in self.index}
+    
+
+def combine_indices(index1, index2):
+    new_index = InvertedIndex(preprocess=False)
+    new_index.index = index1.index
+    for key in index2.index.keys():
+        if key in new_index.index:
+            new_index.index.append(index2.index[key])
+        else:
+            new_index[key] = index2.index[key]
```

Comment:
Looks like it should be:
```suggestion
            new_index.index[key] = index2.index[key]
```
But don't worry I will fix it.